src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,34 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class StringTest {
+    @Test
+    void split() {
+        String[] data = "1,2".split(",");
+        assertThat(data).contains("1");
+        assertThat(data).containsExactly("1", "2");
+    }
+
+    @Test
+    void substring()
+    {
+        String data = "(1,2)".substring(1,4);
+        assertThat(data).isEqualTo("1,2");
+    }
+
+    @DisplayName("Index Error Check")
+    @Test
+    void charAt()
+    {
+        String data = "abc";
+        assertThatExceptionOfType(IndexOutOfBoundsException.class)
+                .isThrownBy(() -> {
+                    data.charAt(3);
```

Comment:
```suggestion
                    "abc".charAt(3);
```

로컬 변수 없이 구현해도 되지 않을가?